install_sct

```diff
@@ -625,21 +625,11 @@ python -m pip install -U "pip!=21.2.*"
 
 ## Install the spinalcordtoolbox into the Conda venv
 print info "Installing Python dependencies..."
```

Comment:
This log message now looks a little weird, since there is just an empty space:

![Screenshot from 2022-01-18 10-26-16](https://user-images.githubusercontent.com/16181459/149966696-8201c659-4cd9-4dcd-be21-419effbe7ee5.png)

We can probably merge the message into the `Installing spinalcordtoolbox...` message so that we just have 1 message that says:

```
Installing spinalcordtoolbox and its Python dependencies...
```